platform/core.startup/src/org/netbeans/core/startup/ConsistencyVerifier.java

```diff
@@ -91,7 +99,21 @@ public static SortedMap<String,SortedSet<String>> findInconsistencies(
                 "org.openide.modules.os.MacOSX, " + // NOI18N
                 "org.openide.modules.os.Linux, " + // NOI18N
                 "org.openide.modules.os.Solaris, " + // NOI18N
-                "org.openide.modules.os.OS2"); // NOI18N
+                "org.openide.modules.os.OS2, " + // NOI18N
+                "org.openide.modules.arch.x86_64, " + // NOI18N
+                "org.openide.modules.arch.amd64, " + // NOI18N
+                "org.openide.modules.arch.x86," + // NOI18N
+                "org.openide.modules.arch.aarch64," + // NOI18N
+                "org.openide.modules.arch.arm," + // NOI18N
+                "org.openide.modules.arch.ppc," + // NOI18N
+                "org.openide.modules.arch.ppc64," + // NOI18N
+                "org.openide.modules.arch.ppc64le," + // NOI18N
+                "org.openide.modules.arch.s390x," + // NOI18N
+                "org.openide.modules.arch.zarch_64," + // NOI18N
+                "org.openide.modules.arch.sparc," + // NOI18N
```

Comment:
Hey @matthiasblaesing , I really don't like that we still support SPARC. It's a long time dead platform.

Also, not a fan of including SPARC64 support. As it's been removed from the JDK for a little while.